packages/nimble-components/src/wafer-map/modules/experimental/hover-handler.ts

```diff
@@ -0,0 +1,79 @@
+import type { WaferMap } from '../..';
+import { PointCoordinates, WaferMapOriginLocation } from '../../types';
+
+/**
+ * HoverHandler deals with user interactions and events like hovering
+ */
+export class HoverHandler {
+    public constructor(private readonly wafermap: WaferMap) {}
+
+    public mousemove(event: MouseEvent): void {
```

Comment:
Reading through the rest of the code, I can't seem to find where this method is being used outside of a test. I feel like I'm missing something.

Reply:
the event handler switches between the stable hover handler, which is the existing one, and the experimental, which is this one. based on which one is selected they get to handle the mousemove event

Reply:
I see...but, as I replied in the other comment, I would prefer we refactor this.